content/english/other_national_portals.md

```diff
@@ -7,11 +7,27 @@ menu:
     weight: 20
 ---
 
+### National Pathogens Portals
+
+In July 2023, the European Molecular Biology Laboratory's European Bioinformatics Institute ([EMBL-EBI](https://www.ebi.ac.uk/)) launched a new version of the [Pathogens Portal](https://www.pathogensportal.org/). This reflected a global move towards general pandemic preparedness, and away from COVID-19 specifically. The Pathogens Portal holds data and information about multiple pathogens and pandemic preparedness in general.
+
+In August 2023, the Swedish Pathogens Portal was launched, becoming the first national pathogens portal node ([see our news on the launch](https://www.pathogens.se/updates/pathogens_portal/)). The Swedish Pathogens Portal was soon used as a basis for other national pathogens portals.
+
+The [Pathogens Data Network (PDN)](https://pathogendatanetwork.org) was formed in 2024. One of its aims was to create a knowledge base enabling access to pathogens data. Work on the knowledge base was co-led by SciLifeLab Data Centre and EMBL-EBI. As part of this work, SciLifeLab Data Centre created [openly available code for a minimal local/national pathogens portal](https://github.com/ScilifelabDataCentre/node-pathogens-portal), based on the Swedish Pathogens Portal, which they developed, to facilitate the launch of other pathogens portals.
+
+#### List of Pathogen Portals
+
+- [Pathogens Portal Switzerland](https://pathogensportal.ch/)
+- [Pathogens Portal Norway](https://pathogens.no/)
+- [Pathogens Portal Netherlands](https://www.pathogensportal.nl/index.html)
```

Comment:
Should we go in alphabetical order ? Like the covid-19 data portal list ?

Also while at it, can we remove the `index.html` from the url, it will look consistent (in the markdown)